bindings.c

```diff
@@ -652,7 +652,9 @@ bool cgfs_remove(const char *controller, const char *cg)
 	if (fd < 0)
 		return false;
 
-	return recursive_rmdir(dirnam, fd);
+	bret = recursive_rmdir(dirnam, fd, cfd);
```

Comment:
Here `cfd` needs to be passed down into `recursive_rmdir()` because `unlinkat()` and `fstatat()` need. `fd` itself is opened in the lines above via `openat(cfd, ...)` and therefore useless for `fstatat()` and `unlinkat()` if I'm not mistaken.